homeworks/07-mars-viewer/src/features/controls/controls.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import {connect} from "react-redux"
+import {AppDispatch, State} from "../../store"
+import {load, selectSol, selectTab} from "../../store/controls/actions"
+import {ControlsTab, Sol} from "../../domain"
+
+type ShowControlsProps = {
+    selectedTab: ControlsTab,
+    selectTab: (tab: ControlsTab) => void,
+    selectedSol: Sol
+    setCurrentSol: (sol: Sol) => void
+    load: (sol: Sol) => void
+}
+
+function ShowControls(props: ShowControlsProps) {
+    return (
+        <div>
```

Comment:
Some padding/margins would help the buttons look better.

Also for navigation between "pages" (photos / favourites) a link would look more intuitive instead of basic buttons.

http://mars-viewer.surge.sh/

![Screenshot 2021-05-25 at 14 00 09](https://user-images.githubusercontent.com/364880/119487004-8e313900-bd61-11eb-87fc-d00fc2d73bdd.png)

---
![Screenshot 2021-05-25 at 14 03 07](https://user-images.githubusercontent.com/364880/119487380-f7b14780-bd61-11eb-9c25-50371ec1528f.png)